docs/design/ovs-pipeline.md

```diff
@@ -604,6 +601,23 @@ is required), as well as for local Pod-to-Pod traffic.
 table=70, priority=0 actions=goto_table:80
 ```
 
+### L3DecTTLTable (71)
+This is the table to decrement TTL for L3 traffic on the source Node. It implements the following functionality:
+   
+ * TTL is decremented by one on the packets which are sent from a local Pod and are destined
+   for a remote Pod. But for the packets that enter OVS pipeline from the local gateway
+   and are destined for a remote Pod, TTL is not decremented in OVS on the source Node. TTL is
+   also decremented on the destination Node. See the flows in [L3ForwardingTable].
+   Traffic then goes to [ConntrackCommitTable], thus skipping [L2ForwardingCalcTable] and
+   the ingress policy rules tables, which are not relevant for traffic destined to a remote
+   Pod (the ingress policy rules will be enforced at the destination Node).
+
+If you dump the flows for this table, you should see something like this:
```

Comment:
Maybe: "you should see flows like the following"

Reply:
Done.